static/js/components/tiles/bar_tile.tsx

```diff
@@ -183,6 +185,11 @@ function processData(
           };
           if (raw.facets[stat.facet]) {
             sources.add(raw.facets[stat.facet].provenanceUrl);
+            let svUnit = raw.facets[stat.facet].unit;
+            if (svUnit && spec.denom) {
```

Comment:
for these, might want to check that "_Person" is in the denominator (we now also have area denominators)

Reply:
done, created a util function